storybook/src/components/Heading/Heading.stories.tsx

```diff
@@ -32,24 +32,24 @@ type Story = StoryObj<typeof meta>
 
 export const Default: Story = {}
 
-export const Heading1: Story = {}
-
-export const Heading2: Story = {
-  args: {
-    level: 2,
-  },
+export const Levels: Story = {
+  render: (args) => (
+    <Column gap="small">
+      {headingLevels.map((level) => (
```

Comment:
I'm not sure I'm a fan of this approach. It makes the code less copy-able for our users.

Reply:
Alright.